.travis.yml

```diff
@@ -1,11 +1,20 @@
+dist: bionic
```

Comment:
Why not focal? That's the latest LTS.

Reply:
I'll update to use `focal` instead, I didn't see it in the documentation.
That should allow to no longer have to rely on `pip` too.